cmd/edit/machinepool/machinepool_test.go

```diff
@@ -36,16 +36,6 @@ var _ = Describe("Machinepool", func() {
 			asBuilder := cmv1.NewMachinePoolAutoscaling().MaxReplicas(2).MinReplicas(0)
 			Expect(builder).To(Equal(asBuilder))
 		})
-
-		It("editMachinePoolAutoscaling should allow 0 min and 0 max replicas", func() {
```

Comment:
There is a validation done in CS that prevents users assigning 0 for max mp for autoscaling.

Reply:
should rosa reports that error before send request to CS side? And then we still need a test case here to verify that

Reply:
Logic looks good, but I feel you might fail test coverage though as lines were deleted.

Reply:
Thanks for your review @marcolan018  @chenz4027 
In addition to those, the validations on the interactive mode are missing. I will create a separate card so that we can have a robust number of test cases to make sure we surface input validation errors from the CLI before reaching CS.

Trying to have this land in RC4 as quickly as possible.
If no concerns, would you kindly approve?

Reply:
@davidleerh I am ok with this

Reply:
Thank you!
Created OCM-8010 for myself to address this.